interfaces/PA-App/src/app/personal-components/select-fsp/select-fsp.component.ts

```diff
@@ -62,13 +62,10 @@ export class SelectFspComponent extends PersonalDirective implements OnInit {
     this.conversationService.startLoading();
     // NOTE: we use this attribute being empty to avoid a new fsp-attribute, while this is a short-term feature only
     this.fsps = this.program.financialServiceProviders.filter(
-      (fsp) => fsp.fspDisplayNamePaApp,
+      (fsp) => fsp.displayName,
```

Comment:
Also check on one of the 2 Intersolve FSP-names that we _do_ want to show...

```suggestion
      (fsp) => fsp.displayName && ['Intersolve-whatsapp', 'Intersolve-paper-voucher'].includes(fsp.fsp),
```

(or similar)